docs/source/admin/quick_howto/cachegroup_fallback_qht.rst

```diff
@@ -0,0 +1,65 @@
+..
+..
+.. _cachegroup-fallback-qht:
+
+******************************
+Configure CacheGroup Fallbacks
```

Comment:
For consistency please use "Cache Group" rather than "CacheGroup"

Reply:
done